### DISCUSSION TITLE
Releitura de Logo

### DISCUSSION THREAD

**Discussion by marcosgarcez** [BrasilAPI/BrasilAPI#218] [Closed] [answered]
Olá pessoal! Essa é a minha primeira issue que abro na vida! 😬
Vi a ideia show do @BrunoS3D e fiz uma releitura pensando na legibilidade e também nas harmonia das cores :D 
Essa foi a ideia dele: https://github.com/BrasilAPI/Logos

A logo tem duas versões, uma horizontal e uma vertical.
Também coloquei aí como ele ficaria monocromática.

![Brasil API](https://user-images.githubusercontent.com/23143043/99616572-491bfb00-29fc-11eb-984b-afb166d7bb8e.png)

O que acharam? 😍

**Comment by filipedeschamps** [comment]
Wow!!! 😳😳😳 ficou SENSACIONAL!!! Cores e tipografia 100% lindas!

Você se importaria em fazer a mesma releitura do logo que eu fiz? Eu
adoraria ver ele sob a ótica de alguém que sabe o que ta fazendo com design
😂

**Comment by brunos3d** [comment]
Que delicinhaaaaa!!! 😍

Concordo com o Deschamps, acho que uma releitura da logo seria muito interessante! 😁
O minimalismo das formas combinado com seu estilo de criação, o que acha mestre?

**Comment by arrudagates** [comment]
Nossa, esse logo ficou muito bom, eu já teria aprovado sem pensar duas vezes.

**Comment by RodriAndreotti** [comment]
Ficou filé demais o logo. :-p

**Comment by marcosgarcez** [comment]
Opa, seria uma honra fazer uma releitura @filipedeschamps! 😄

**Comment by LucasMarcolino237** [comment]
Muito bom, achei genial esse logo, ficou realmente muito bom!

**Comment by marcosgarcez** [comment]
Olá pessoal!  👋 

A pedido do @filipedeschamps também fiz uma releitura do logo que ele fez.

Nessa releitura, pensei em um símbolo quadrado, pois ele consegue se adequar bem em vários lugares, principalmente como favicon. 
Também tentei resolver alguns problemas de legibilidade do logo, como o espaçamento entre as formas, pois estava pequeno e dependendo do tamanho em que o logo ficava o espaçamento acabava "sumindo".

Mantive quase todas as cores da paleta da outra releitura, apenas com uma leve alteração no tom de azul.

Todo o logo foi construído utilizando a proporção áurea a fim de manter uma harmonia e padronizar o processo de construção 😄 

![Brasil API 2](https://user-images.githubusercontent.com/23143043/99861473-e601a400-2b74-11eb-967a-28a869f135e9.png)

Esse é o grid de construção:

![Construcao Brasil API 2](https://user-images.githubusercontent.com/23143043/99861479-f0bc3900-2b74-11eb-997d-d8f4b1c30bcf.png)

**Comment by filipedeschamps** [comment]
Ahhhhh que massaaaaaa 😍 😍 😍 😍 e que maravilha ver isso feito por um profissional 🤝 

No original eu também tinha usado o golden ratio, sou **viciado** nele, e queria representar que a API era composta por infinitas partes proporcionais, onde na sua maior simplicidade possível (de abstração) eram as 3 que estavam no logo.

Não tive a sacada de fazer ele quadrado, muito bem bolado e de fato vai se enxaicar em muitos lugares, incluindo no ícone da Org... quando eu fui atualizar com o logo, o espaço lá era quadrado, daí tive que cortar kkkk 😂 

Muito obrigado pela contribuição, matou a pau!!! 👍

**Comment by marcosgarcez** [comment]
> Ahhhhh que massaaaaaa 😍 😍 😍 😍 e que maravilha ver isso feito por um profissional 🤝
> 
> No original eu também tinha usado o golden ratio, sou **viciado** nele, e queria representar que a API era composta por infinitas partes proporcionais, onde na sua maior simplicidade possível (de abstração) eram as 3 que estavam no logo.
> 
> Não tive a sacada de fazer ele quadrado, muito bem bolado e de fato vai se enxaicar em muitos lugares, incluindo no ícone da Org... quando eu fui atualizar com o logo, o espaço lá era quadrado, daí tive que cortar kkkk 😂
> 
> Muito obrigado pela contribuição, matou a pau!!! 👍

É uma honra contribuir com um projeto **massinha** como esse! ❤️ 😄 🤝 

Estou disponibilizando aqui os arquivos em SVG do logo 👇 

[Logo Brasil API.zip](https://github.com/BrasilAPI/BrasilAPI/files/5576781/Logo.Brasil.API.zip)

Qualquer coisa estou a disposição 😉 
Vlw!

**Comment by raymag** [comment]
> 
> 
> Olá pessoal! 👋
> 
> A pedido do @filipedeschamps também fiz uma releitura do logo que ele fez.
> 
> Nessa releitura, pensei em um símbolo quadrado, pois ele consegue se adequar bem em vários lugares, principalmente como favicon.
> Também tentei resolver alguns problemas de legibilidade do logo, como o espaçamento entre as formas, pois estava pequeno e dependendo do tamanho em que o logo ficava o espaçamento acabava "sumindo".
> 
> Mantive quase todas as cores da paleta da outra releitura, apenas com uma leve alteração no tom de azul.
> 
> Todo o logo foi construído utilizando a proporção áurea a fim de manter uma harmonia e padronizar o processo de construção 😄
> 
> ![Brasil API 2](https://user-images.githubusercontent.com/23143043/99861473-e601a400-2b74-11eb-967a-28a869f135e9.png)
> 
> Esse é o grid de construção:
> 
> ![Construcao Brasil API 2](https://user-images.githubusercontent.com/23143043/99861479-f0bc3900-2b74-11eb-997d-d8f4b1c30bcf.png)

Cara, ficou show! Parabéns pelo trabalho.

**Comment by RodriAndreotti** [comment]
Eu diria mais, diria até que gostei mais da versão quadrada do que da versão com o PI

**Comment by AlanMartines** [comment]
Sensacional a logo...

**Comment by fabmont** [comment]
Uaaaaau! Que proposta incrível, @marcosgarcez ! 😁 😍 👏
Ficou muito show da releitura da logo que já vem sendo usada!

**Comment by lucianopf** [comment]
😱 ❤️ 😍 🚀 

CARAAAACA, SENSACIONAL! 🤩 

Achei as 2 muuuito legais!!!

**Comment by Caaddss** [comment]
Nossa q lindo!!!!!

Eu amei

**Comment by brunofamiliar** [comment]
Ficou show essa releitura!! 🥇 😄

**Comment by marcosgarcez** [comment]
Olá pessoal! 👋
Fiz um pull request #251 com o último logo feito! 😃

**Comment by lucianopf** [comment, marked as answer]
Aeeee!! Muito obrigado @marcosgarcez PR aprovado e mergeado na master!! ❤️ 🚀 👏 🎉
